posthog/hogql/database/schema/test/test_channel_type.py

```diff
@@ -549,3 +549,19 @@ def test_duckduckgo_paid_click(self):
     #             "https://l.facebook.com/",
     #         ),
     #     )
+
+    def test_zendesk_ticket_14945(self):
```

Comment:
I'm wondering wether we should move the regression conditions inline or into a GitHub ticket, as this will introduce another source of truth

Reply:
What do you mean by moving them inline?

Reply:
Just adding a bit of text explaining into what issue we ran here so that people can understand the test even if Zendesk isn't around any more.